.github/workflows/bumpversion.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     name: Bump version and push tags to master
     steps:
       - name: Bump version
-        uses: Clinical-Genomics/bump2version-ci@master
+        uses: Clinical-Genomics/bump2version-ci@3
```

Comment:
👍

Reply:
Just checking since we had it for `setup.py` - I assume this work ok with the `__version__` in the module init as well?

Reply:
Yes, the bumpversion will update the version.